T-Dict/T-Dict/Controller/SearchVC/SearchHistoryCell/SearchHistoryCell.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  SearchHistoryCell.swift
+//  T-Dict
+//
+//  Created by Tiến on 3/26/21.
+//  Copyright © 2021 Tiến. All rights reserved.
+//
+
+import UIKit
+
+class SearchHistoryCell: UITableViewCell {
+    
+    static let identifier = "SearchHistoryCell"
+    @IBOutlet weak var contentLabel: UILabel!
+    @IBOutlet weak var deleteButton: UIButton!
+    @IBOutlet weak var selectButton: UIButton!
+
+    static func nib() -> UINib{
```

Comment:
1. coding convension
2. cái này hoàn toàn có thể tách ra extension common rồi dùng Generic
3. có thể dùng lib Reusable để xử lý thay cho option thứ 2